daemon/response.go

```diff
@@ -150,7 +151,7 @@ func ErrorResponse(status int) ErrorResponseFunc {
 		Status: status,
 	}
 
-	return r.SetError
+	return r.setError
```

Comment:
ErrorResponder ?  ErrorResponseMaker? it seems it's actually never used out response.go itself so not sure it needs a super cute name